opus/application/static_media/js/browse.js

```diff
@@ -1295,7 +1401,7 @@ var o_browse = {
         //   it for the case when cached data is removed so that scrollbar position is always correct (and never reaches to the
         //   end until it reaches to the end of the data)
         o_browse.setScrollbarPosition(infiniteScrollData.options.sliderObsNum,
```

Comment:
I think you can simplify the code a little bit here if you make infiniteScrollData.options a variable.